### PR TITLE
Byandell posts/01-first-map almost!

### PR DESCRIPTION
**Pull request by byandell**
See my comment on https://github.com/earthlab-education/Earth-Analytics-AY24/issues/169.

Basically, I did the task, but could not figure out how to create a folder in the posts/01-first-map subfolder, so my files are just laying around. Advice?

### CODE REVIEW

**Comment by byandell** [comment]
I redid this, creating a folder byandell and put in files uwmadison.html and uwmadison.md

File: posts/01-first-map/byandell/uwmadison.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# UW-Madison
+
+[UW-Madison](https://wisc.edu) is located in Madison, WI.
+There are 40k+ students and thousands of faculty and staff.
+Here is the
+[campus map](https://map.wisc.edu)
+and below is an interactive image of the campus area.
+
+<embed type="text/html" src="osm.html" width="600" height="600">
```
Comment (lauren-alexandra) [inline posts/01-first-map/byandell/uwmadison.md:9]: The markdown looks good but I would update the src path from "osm.html" to "uwmadison.html"

**Review by lauren-alexandra — CHANGES_REQUESTED** [review]
File path needs to be updated in <embed> in uwmadison.md to "uwmadison.html"

File: posts/01-first-map/byandell/uwmadison.md (hunk 1 above)
Comment (byandell) [reply posts/01-first-map/byandell/uwmadison.md]: Done. Thanks for catching this.

**Review by lauren-alexandra — APPROVED** [review]

**Comment by eculler** [comment]
Sweet @byandell looks like you figured this out!

**Pull request merged**